microsetta_public_api/server.py

```diff
@@ -32,13 +30,22 @@ def build_app(resources_config_json=None):
 if __name__ == "__main__":
     import sys
     config_fp = sys.argv[1] if len(sys.argv) > 1 else None
-    PORT = 8084
+
+    if config_fp is None:
+        resource_config = None
+        server_config = {}
+    else:
+        with open(config_fp) as fp:
+            server_config = json.load(fp)
+        resource_config = config['resources']
```

Comment:
```suggestion
        resource_config = server_config.get('resources', None)
```

Reply:
This will allow the `JSON` to actually affect the dictionary, as opposed to trying to fill the dictionary with the contents that are already in it.